ReactiveDataDisplayManagerTests/REFACTORING/Table/Mock/View/UITableViewSpy.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UITableViewSpy.swift
+//  ReactiveDataDisplayManagerTests
+//
+//  Created by Anton Eysner on 08.02.2021.
+//  Copyright © 2021 Александр Кравченков. All rights reserved.
+//
+
+import UIKit
+
+final class UITableViewSpy: UITableView {
```

Comment:
```suggestion
final class SpyUITableView: UITableView {
```

Чтобы с другими Stub/Mock/Spy было однородно.